src/main/java/net/minecraftforge/fml/common/Loader.java

```diff
@@ -114,6 +114,8 @@
 public class Loader
 {
     public static final String MC_VERSION = net.minecraftforge.common.ForgeVersion.mcVersion;
+    private static final ImmutableList<String> DEPENDENCYINSTRUCTIONS = ImmutableList.of("client", "server", "required", "before", "after");
```

Comment:
`DEPENDENCY_INSTRUCTIONS`
Although other constant fields' names are not cased properly, I suggest you to make them proper.